src/Juvix/Backends/LLVM/Codegen/Block.hs

```diff
@@ -200,8 +227,8 @@ externf ∷ Externf m ⇒ Name → m Operand
 externf name = getvar (nameToSymbol name)
 
 nameToSymbol ∷ Name → Symbol
-nameToSymbol (UnName n) = (intern (show n))
-nameToSymbol (Name n) = (intern (show n))
+nameToSymbol (UnName n) = (intern (filter (/= '\"') (show n)))
```

Comment:
Why no quotes? Are they disallowed? Maybe we should check this when `Name`s are created.